spec/api-app-spec.js

```diff
@@ -289,4 +289,37 @@ describe('app module', function () {
       })
     })
   })
+
+  describe('app.launcher API', function() {
+    it('should be available on linux', function() {
+      if (process.platform !== 'linux') {
+        assert.equal(app.launcher, undefined);
```

Comment:
There was previously an issue where `npm run lint` did not properly link the `spec/` folder.

This was fixed in #6289 so if you rebase or merge master into this branch and then run `npm run lint`, you should see some errors for this file like the `;` at the end of this line.

Sorry that this wasn't picked up sooner.